src/Pages/Withdrawals.razor

```diff
@@ -722,22 +717,22 @@
 
     private async Task RejectOrCancel()
     {
-        if ((_rejectCancelStatus == WalletWithdrawalRequestStatus.Cancelled || _rejectCancelStatus == WalletWithdrawalRequestStatus.Rejected) 
```

Comment:
did you perform these formatting changes on the go or  were they automatically done by your IDE? We should keep merge requests small and change only what it is described in the issue IMHO.
We can always create a new issue to address formatting issue

Reply:
Done by the IDE, MB I didnt payed attention to it